### PR TITLE
Add danny

### PR DESCRIPTION
**Pull request by 8ugsBunny**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -45,7 +45,7 @@
 	
 
 	
-	<div id="left-panel"><!--左側鑲版-->
+	<div id="left-panel" class="ggbaba"><!--測試--><!--左側鑲版-->
```
Comment (vince7ben23) [inline index.php:48]: you did shitty

File: index.php [hunk 2]
```diff
@@ -45,7 +45,7 @@
 	
 
 	
-	<div id="left-panel"><!--左側鑲版-->
```
Comment (8ugsBunny) [inline index.php:48]: I am fucking GENIUS

**Review by 8ugsBunny — COMMENTED** [review]
This code's author is fucking AWESOME

File: index.php (hunk 2 above)
Comment (8ugsBunny) [inline index.php:48]: Hello Mr.Lin. Would you hit me ?

**Pull request merged**